ConnectionsPanel.js

```diff
@@ -420,4 +382,110 @@ ToolsButton.propTypes = {
   count:         PropTypes.number,
 };
 
+const ConnectionsPanelSection = ({ title, children }) => {
+  const { themeStr, interfaceLanguage } = useContext(GlobalStateContext);
+  const theme = getTheme(themeStr);
+  return (
+    <View style={styles.connectionPanelSection}  flexdir={interfaceLanguage === "english" ? null : styles.rtlRow}>
+      {title &&
+        <View style={styles.connectionPanelTitle}>
+          <Text style={[interfaceLanguage === "hebrew" ? styles.heInt : styles.enInt, theme.tertiaryText]}>
+            {title}
+          </Text>
+        </View>
+      }
+      <View>
+        {children}
+      </View>
+    </View>
+  );
+};
+ConnectionsPanelSection.propTypes = {
+  title: PropTypes.string,
+  children: PropTypes.node.isRequired,
+};
+
+const TopButtons = ({relatedHasError, reloadRelated, themeStr, setConnectionsMode, sheet, versionsCount}) => {
+  return (
+    <ConnectionsPanelSection>
+      {!relatedHasError ? null : <ToolsButton
+        text={strings.resourcesFailedToLoad}
+        onPress={reloadRelated}
+      /> }
+      <ToolsButton
+        text={strings.aboutThisText}
+        icon={iconData.get('info', themeStr)}
+        onPress={()=> {setConnectionsMode("about"); }}
+      />
+      {sheet ? null : <ToolsButton
+        text={strings.translations}
+        icon={iconData.get('translations', themeStr)}
+        count={versionsCount}
+        onPress={()=> {setConnectionsMode("versions"); }}
+      /> }
+    </ConnectionsPanelSection>
+  )
+};
+TopButtons.propTypes = {
+    themeStr:           PropTypes.string.isRequired,
+    setConnectionsMode: PropTypes.func.isRequired,
+    versionsCount:      PropTypes.number.isRequired,
+    relatedHasError:    PropTypes.bool,
+    reloadRelated:      PropTypes.func.isRequired,
+  // what is sheet?
```

Comment:
this is a good question. It's a bad habit from a long time ago. Sheet is the full sheet object, assuming you're viewing a sheet. We often use it as a boolean to say "are you currently viewing a sheet?". This is very bad practice...